src/modm/board/module.lb

```diff
@@ -17,6 +17,16 @@ def init(module):
 def prepare(module, options):
 	# The modm_abandon function uses delay functions
     module.depends(":architecture:delay", ":architecture:assert")
+
+    # Backwards compatible move, remove this in 2022.
+    module.add_alias(Alias(name="blue-pill", destination="blue-pill-f103",
+                           description="Renamed for clarity."))
+    module.add_alias(Alias(name="black-pill", destination="black-pill-f103",
+                           description="Renamed for clarity."))
+    module.add_alias(Alias(name="mini-f401", destination="black-pill-f401",
+                           description="Renamed for clarity."))
+    module.add_alias(Alias(name="mini-f411", destination="black-pill-f411",
+                           description="Renamed for clarity."))
```

Comment:
Aren't the aliases already added in `repo.lb` above?

Reply:
Yes, but for the Config nodes (`modm:mini-f411` for board.xml), these are for the board module nodes (`modm:board:mini-f411` for board.hpp).

Reply:
Ah, missed that.